static/app/components/group/externalIssueForm.tsx

```diff
@@ -33,7 +32,7 @@ export default class ExternalIssueForm extends AbstractExternalIssueForm<Props,
   loadTransaction?: ReturnType<typeof Sentry.startTransaction>;
   submitTransaction?: ReturnType<typeof Sentry.startTransaction>;
 
-  componentDidMount() {
+  componentWillMount() {
```

Comment:
@ceorourke Is there any way to do this without using `componentWillMount` since it's considered a legacy and unsafe method?

Reply:
Why not keep `DidMount`?

Reply:
@billyvg idk why but it reverts to the non-waterfall view that way. I just tried putting it in a constructor and that seems to work fine but I'm not sure that's the best place.

Reply:
Ah, actually that makes sense, IIRC `WillMount` gets called somewhere between `constructor` and `DidMount` -- +1 to putting it in the constructor.